runtime/src/main/java/io/quarkiverse/mybatis/runtime/MyBatisRuntimeConfig.java

```diff
@@ -31,4 +31,11 @@
      */
     @ConfigItem(name = "initial-sql")
     public Optional<String> initialSql;
+
+    /**
+     * MyBatis initial sql
+     */
```

Comment:
Please change this comment to reflect the right option. Thanks !

Reply:
yes